contracts/gov/GovPool.sol

```diff
@@ -300,6 +314,33 @@ contract GovPool is
         emit Delegated(msg.sender, delegatee, amount, nftIds, true);
     }
 
+    function delegateTreasury(
+        address delegatee,
+        uint256 amount,
+        uint256[] calldata nftIds
+    ) external override onlyThis {
+        require(amount > 0 || nftIds.length > 0, "Gov: empty delegation");
+
+        require(dexeExpertNft.isExpert(delegatee), "Gov: delegatee is not an expert");
+
+        if (amount != 0) {
+            IERC20(_govUserKeeper.tokenAddress()).transfer(delegatee, amount);
```

Comment:
```suggestion
            IERC20(_govUserKeeper.tokenAddress()).transfer(_govUserKeeper, amount);
```

Reply:
If you are doing the decimals conversion on undelegation, then you have to do it on delegation as well